magefile.go

```diff
@@ -2792,7 +2792,7 @@ func authGCP(ctx context.Context) error {
 	}
 
 	// Check the authenticated account's project
-	cmd = exec.CommandContext(ctx, cliName, "config", "get", "core/project")
+	cmd = exec.CommandContext(ctx, cliName, "config", "get-value", "core/project")
```

Comment:
I wonder why this hasn't failed for others on the team.  @michel-laterman What version of the `gcloud` CLI are you using?  For comparison sake, here's what I'm using:

```
$ gcloud version
Google Cloud SDK 477.0.0
bq 2.1.4
core 2024.05.17
gcloud-crc32c 1.0.0
gke-gcloud-auth-plugin 0.5.8
gsutil 5.29
```

Reply:
If it turns out that this change is due to a difference in the version of `gcloud` we should either make this change conditional on the version or document the suggested `gcloud` CLI version constraints in https://github.com/elastic/elastic-agent/blob/main/docs/test-framework-dev-guide.md#dependencies.